phase1/gce/configure-vm-kubeadm.sh

```diff
@@ -3,30 +3,52 @@
 TOKEN=$(get_metadata "k8s-kubeadm-token")
 KUBEADM_VERSION=$(get_metadata "k8s-kubeadm-version")
 KUBERNETES_VERSION=$(get_metadata "k8s-kubernetes-version")
-
+KUBELET_VERSION=$(get_metadata "k8s-kubelet-version")
 curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
 
-if [[ "${KUBEADM_VERSION}" == stable ]]; then
+if [[ "${KUBELET_VERSION}" == stable ]]; then
```

Comment:
I would expect this check to be for KUBELET_VERSION or KUBEADM_VERSION to be set to stable, in order to add the Kubernetes repo source. Then, you can individually do `apt-get install -y kubelet` and/or `apt-get install -y kubeadm` based on which one is set to stable.

Reply:
As noted in the other comment, I cannot use a package update/install of kubeadm if it differs from the kubelet.